.github/workflows/Build-and-deploy-reporting-services.yaml

```diff
@@ -6,107 +6,107 @@ on:
-    with:
-      microservice_name: investigation-reporting-service
-      dockerfile_relative_path: -f ./investigation-service/Dockerfile .
-      environment_classifier: SANDBOX
-      java_version: "17"
-    secrets:
-      CDC_NBS_SANDBOX_SHARED_SERVICES_ACCOUNTID: ${{secrets.CDC_NBS_SANDBOX_SHARED_SERVICES_ACCOUNTID}}
-      ECR_REPO_BASE_NAME: ${{secrets.ECR_REPO_BASE_NAME}}
-      GIT_USER_EMAIL: ${{secrets.GIT_USER_EMAIL}}
-      GIT_USER_NAME: ${{secrets.GIT_USER_NAME}}
-      HELM_TOKEN: ${{secrets.HELM_TOKEN}}
-    
-  # post-processing-reporting-microservice
-  call-build-post-processing-reporting-microservice-container-workflow:
-    name: Build post-processing Reporting Service Container
-    uses: CDCgov/NEDSS-Workflows/.github/workflows/Build-gradle-microservice-container.yaml@main
-    with:
-      microservice_name: post-processing-reporting-service
-      dockerfile_relative_path: -f ./post-processing-service/Dockerfile .
-      environment_classifier: SANDBOX
-      java_version: "17"
-    secrets:
-      CDC_NBS_SANDBOX_SHARED_SERVICES_ACCOUNTID: ${{secrets.CDC_NBS_SANDBOX_SHARED_SERVICES_ACCOUNTID}}
-      ECR_REPO_BASE_NAME: ${{secrets.ECR_REPO_BASE_NAME}}
-      GIT_USER_EMAIL: ${{secrets.GIT_USER_EMAIL}}
-      GIT_USER_NAME: ${{secrets.GIT_USER_NAME}}
-      HELM_TOKEN: ${{secrets.HELM_TOKEN}}
-
-  # observation-reporting-microservice
-  call-build-observation-reporting-microservice-container-workflow:
-    name: Build Observation Reporting Service Container
-    uses: CDCgov/NEDSS-Workflows/.github/workflows/Build-gradle-microservice-container.yaml@main
-    with:
-      microservice_name: observation-reporting-service
-      dockerfile_relative_path: -f ./observation-service/Dockerfile .
-      environment_classifier: SANDBOX
-      java_version: "17"
-    secrets:
-      CDC_NBS_SANDBOX_SHARED_SERVICES_ACCOUNTID: ${{secrets.CDC_NBS_SANDBOX_SHARED_SERVICES_ACCOUNTID}}
-      ECR_REPO_BASE_NAME: ${{secrets.ECR_REPO_BASE_NAME}}
-      GIT_USER_EMAIL: ${{secrets.GIT_USER_EMAIL}}
-      GIT_USER_NAME: ${{secrets.GIT_USER_NAME}}
-      HELM_TOKEN: ${{secrets.HELM_TOKEN}}
-
- # LdfData-reporting-microservice
-  call-build-ldfdata-reporting-microservice-container-workflow:
-    name: Build LdfData Reporting Service Container
-    uses: CDCgov/NEDSS-Workflows/.github/workflows/Build-gradle-microservice-container.yaml@main
-    with:
-      microservice_name: ldfdata-reporting-service
-      dockerfile_relative_path: -f ./ldfdata-service/Dockerfile .
-      environment_classifier: SANDBOX
-      java_version: "17"
-    secrets:
-      CDC_NBS_SANDBOX_SHARED_SERVICES_ACCOUNTID: ${{secrets.CDC_NBS_SANDBOX_SHARED_SERVICES_ACCOUNTID}}
-      ECR_REPO_BASE_NAME: ${{secrets.ECR_REPO_BASE_NAME}}
-      GIT_USER_EMAIL: ${{secrets.GIT_USER_EMAIL}}
-      GIT_USER_NAME: ${{secrets.GIT_USER_NAME}}
-      HELM_TOKEN: ${{secrets.HELM_TOKEN}}
+#  call-build-person-reporting-microservice-container-workflow:
```

Comment:
Removing the github actions to build docker image from this repo for the RTR services as we have moved them to the new repo.